### PR TITLE
Make sure to set a UserLanguage otherwise Google Analytics won't pick up the trackings

### PR DESCRIPTION
**Pull request by GeertvanHorrik**
… up the trackings

### CODE REVIEW

File: GoogleAnalyticsTracker.Core/TrackerBase.async.cs [hunk 1]
```diff
@@ -63,6 +63,12 @@ private void SetRequired(IGeneralParameters parameters)
         {
             parameters.TrackingId = TrackingAccount;
             parameters.ClientId = AnalyticsSession.GenerateSessionId();
+
+            if (string.IsNullOrWhiteSpace(parameters.UserLanguage))
+            {
+                // Note: another way could be CultureInfo.CurrentCulture
```
Comment (maartenba) [inline GoogleAnalyticsTracker.Core/TrackerBase.async.cs]: Using CurrentCulture is perhaps the nicer way to go?

Comment (GeertvanHorrik) [inline GoogleAnalyticsTracker.Core/TrackerBase.async.cs]: Agree. I am not a real asp.net expert so not sure if the CurrentCulture represents the server culture or takes the culture of the request.

Comment (maartenba) [inline GoogleAnalyticsTracker.Core/TrackerBase.async.cs]: Typically the server language, but if well developed it would be the client's culture :)

**Comment by maartenba** [comment]
Thanks @GeertvanHorrik

**Pull request merged**